### ISSUE TITLE
Scaling fluentd-events pods?

### ISSUE THREAD

**Issue by knguyenst** [SumoLogic/sumologic-kubernetes-collection#2380]
There isn't an option to set replicaCount or autoscaling for fluentd-events pods via helm chart.

Is this by design?

**Comment by swiatekm-sumo** [comment]
Yes, this is by design. Collecting events is essentially reading records from a database table - if you want to scale that, you need to shard the table somehow, which is possible, but involves significant additional complexity.

Do you need to scale the event collection? Even very large clusters, a single FluentD Pod should be able to cope with the load, and if not, the equivalent we're introducing using the Opentelemetry Collector(https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/2379) should do the trick in the near future.

**Comment by astencel-sumo** [comment]
How long does it take for a new events collector pod to be created on another node after previous node went down? Perhaps adjusting your alerting is enough?

We are reluctant to add a high-availability feature to the events collector (currently it is Otelcol, not Fluentd), as it increases complexity.
